src/main/java/com/feelmycode/parabole/repository/EventRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.feelmycode.parabole.repository;
+
+import com.feelmycode.parabole.domain.Event;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface EventRepository extends JpaRepository<Event, Long> {
+    List<Event> findAllByIsDeleted(boolean isDeleted);
```

Comment:
클래스 내부 첫줄은 통일되게 띄워주세요~
```suggestion
public interface EventRepository extends JpaRepository<Event, Long> {

    List<Event> findAllByIsDeleted(boolean isDeleted);
```

Reply:
수정했습니다~